account_invoice_report_hide_line/README.rst

```diff
@@ -70,6 +71,7 @@ Contributors
 ~~~~~~~~~~~~
 
 * Bhavesh Odedra <bodedra@opensourceintegrators.com>
+* Raphael Lee <rlee@opensourceintegrators.com>
```

Comment:
@RLeeOSI You need to update the files in the readme folder. This file is generated from there so your changes will be overwritten.